contents/tutorials/power-users.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How to identify and analyze power users
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-10-26
+featuredImage: ../images/tutorials/banners/power-users.png
+topics: ['trends', 'cohorts']
+---
+
+Power users are users who use your product the most, significantly more than the average user. They have different needs and usage patterns than average users. Often, they are a company’s best and most important customers.
+
+Power users are knowledgeable, willing to learn, and likely to give feedback about your product. They care more about new features and upgrades than the average user does. Changes can impact a power user while not impacting an average user. That is why it is important to analyze power users and build for them as a group separately.
+
+Building a better product for power users keeps them happy and helps prevent them from churning. It helps you build a competitive advantage by growing lock-in with these users. It also  encourages average users to increase their usage and become power users.
+
+To create a great product for power users, you must know who they are and how they are using your product. In this tutorial, we’ll cover how to identify and analyze power users. All you need to start is a PostHog instance (and some users).
+
+## Defining your power users
+
+The first step to identifying and analyzing a power user is defining them. There is no one-size-fits-all, every company and product has its own definition. Defining what they do is key to identifying and analyzing them. Ideally, your power users should align with your ideal customer profile and your [key product metrics](/blog/b2b-saas-product-metrics.
```

Comment:
```suggestion
There is no one-size-fits-all for a power users – every company and product has its own. Defining what they do is key to identifying and analyzing them. Ideally, your power users should align with your ideal customer profile and your [key product metrics](/blog/b2b-saas-product-metrics.
```

RE: product metrics, couldn't you argue your product metrics should align to your power users actions, rather than the other way around?

Reply:
I think it works in both direction. Power users align with metrics, metrics align with power users.